gem/static/js/springster.js

```diff
@@ -113,15 +113,14 @@
   };
 
   var loaderAnimation = function() {
-    var wrapper = document.getElementById("content-wrapper")
-    
-    wrapper.addEventListener("click", function(event){
-      var element = event.target;
-      if (element.classList.contains("call-to-action__button--primary")) {
-        element.setAttribute('value', '');
-        element.innerHTML = '';
-        element.classList.add('call-to-action__button--loader');
-       }
+    var ctaBtn = document.querySelectorAll('.call-to-action__button--primary');
+
+    Array.from(ctaBtn).forEach(link => {
```

Comment:
Are we transpiling our JS? Just because `=>` may not work for some older browsers.